### ISSUE TITLE
Clippy warning

### ISSUE THREAD

**Issue by Yamakaky** [brson/error-chain#51]
```
warning: redundant closure found, #[warn(redundant_closure)] on by default
  --> src/types/decode.rs:5:1
   |
5  | error_chain! {
   | ^
   |
help: remove closure as shown:
   | . unwrap_or_else ( $ crate :: make_backtrace ) ; $ error_name (
   = help: for further information visit https://github.com/Manishearth/rust-clippy/wiki\#redundant_closure
   = note: this error originates in a macro outside of the current crate
```
Seems to be https://github.com/brson/error-chain/blob/master/src/lib.rs#L503, but I can't find why...

**Comment by Arnavion** [comment]
It was already fixed in #33 which is why you can't find why. Not released yet though.

**Comment by Yamakaky** [comment]
Oh, cool. Any idea about when?
